src/rasenmaeher_api/web/api/product/views.py

```diff
@@ -138,3 +140,29 @@ async def get_kc_token(
         raise HTTPException(403, detail="KC is not enabled")
     data = await KCClient.singleton().client_access_token()
     return KCClientToken.parse_obj(data)
+
+
+@router.post("/interop/{tgtproduct}", dependencies=[Depends(MTLSHeader(auto_error=True))])
+async def add_interop(
+    srcproduct: ProductAddRequest,
+    tgtproduct: str,
+    request: Request,
+) -> OperationResultResponse:
+    """Product needs interop privileges with another"""
+    payload = request.state.mtlsdn
+    if payload.get("CN") not in RMSettings.singleton().valid_product_cns:
+        raise HTTPException(status_code=403)
+
+    # TODO: Verify that srcproduct certcn and actual cert contents match
+
+    manifest = RMSettings.singleton().kraftwerk_manifest_dict
+    if "products" not in manifest:
+        LOGGER.error("Manifest does not have products key")
+        raise HTTPException(status_code=500, detail="Manifest does not have products key")
+    if not tgtproduct in manifest["products"]:
```

Comment:
not in

Reply:
https://github.com/pvarki/python-rasenmaeher-api/pull/124/commits/95b84cb00311e9972288641f5a7715aed8f99a83